blueocean-dashboard/src/main/js/components/ScrollToHash.jsx

```diff
@@ -5,7 +5,7 @@ export const scrollToHash = ComposedComponent => class extends Component {
     componentDidMount() {
         setTimeout(() => {
             this.scrollToAnchor();
-        });
+        }, 100);
```

Comment:
I changed it to 1, but basically I need to give time for ResultItem to expand. further setTimeout should have the second parameter anyway